runBiDiServer.sh

```diff
@@ -19,7 +19,13 @@ if [[ $# -gt 0 && ("$1" == "-h" || "$1" == "--help") ]]; then
 fi
 
 # The chrome release channel to use: `stable`, `beta`, `canary`, `dev`.
-readonly CHANNEL="${CHANNEL:-dev}"
+CHANNEL="${CHANNEL:-local}"
+
+# Run the pinned CfT Cannary by default
+if [ $CHANNEL == "local" ]; then
+  export BROWSER_BIN=$(node install-browser.mjs ~/.cache/chromium-bidi --shell)
```

Comment:
It would be cleaner if we did not hard-code any cache in this file. Can we omit it?

Instead, either keep using `cwd`, or update `install-browser.mjs` to use `.cache`, that's where it should belong.

Reply:
OK i will take a stab, but it will require some more changes.